.gitignore

```diff
@@ -1,14 +1,3 @@
-# Don’t commit the following directories created by pub.
-.buildlog
-.pub/
-build/
+.pub
 packages
-
-# Or the files created by dart2js.
-*.dart.js
-*.js_
-*.js.deps
-*.js.map
-
-# Include when developing application packages.
```

Comment:
I'd keep this file as it was. The test do run pub-build, so if we are testing locally it's nice to not have to delete all the generated files